src/components/icons/SuccessIcon.tsx

```diff
@@ -5,8 +5,8 @@ export function SuccessIcon(props: any) {
   return (
     <Icon
       viewBox="0 0 150 150"
-      width={props.width ?? '150'}
-      height={props.height ?? '150'}
+      width="150"
```

Comment:
I'm pretty confident with chakra we can use the property `boxSize={25}` on the icon instance to make the size different than the default. And we are not required to pass in the width and height through props.